connectors/aws-cert/src/main/service/rhel/files/default-conf/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+connector_name='aws-cert'
```

Comment:
All of the start.sh files except airwatch-connector's are the same except this 1 connector_name variable.